### PR TITLE
General improvements

### PR DESCRIPTION
**Pull request by adityasaky**
:)

### CODE REVIEW

**Comment by adityasaky** [comment]
cc @petgoldfish

File: src/pages/index.tsx [hunk 1]
```diff
@@ -20,21 +20,22 @@ function IndexPage() {
...
 			/>
-			<h1>Hello World!</h1>
+			<h1>Hello, world!</h1>
 			<p>
-				I live in New York City, pursuing a Master's in Computer Science at NYU.
-				Prior to this, I was a Full Stack Software Engineer at PowerSchool in
-				Bangalore, India.
+				I live in New York City, and I'm pursuing a Master's in Computer
+ 				Science at NYU. Prior to this, I was a Full Stack Software
+ 				Engineer at PowerSchool in Bangalore, India.
```
Comment (petgoldfish) [inline src/pages/index.tsx:27]: no

File: src/pages/index.tsx [hunk 2]
```diff
@@ -20,21 +20,22 @@ function IndexPage() {
...
 			</p>
 			<p>
 				When I'm not playing around with Web Development or Programming
-				Languages, I'm usually playing video games or doing outdoorsy stuff like
-				hiking and traveling the world (well, before the pandemic at least)!
+				Languages, I'm usually playing video games or doing outdoorsy
+                stuff like hiking and traveling the world (well, before the
+                pandemic at least)!
```
Comment (petgoldfish) [inline src/pages/index.tsx:33]: no

File: src/pages/index.tsx [hunk 3]
```diff
@@ -20,21 +20,22 @@ function IndexPage() {
...
 			</p>
 			<p>
 				Check out my{" "}
 				<a href="/resume" target="_blank" rel="noopener noreferrer">
-					resume
+					résumé
```
Comment (petgoldfish) [inline src/pages/index.tsx:38]: I approve

File: src/pages/portfolio.tsx [hunk 4]
```diff
@@ -117,7 +117,7 @@ function PortfolioPage() {
 			</article>
 			<hr />
 			<article>
-				<h2 className="project-type">OTHERS</h2>
+				<h2 className="project-type">MISC.</h2>
```
Comment (petgoldfish) [inline src/pages/portfolio.tsx:120]: A+

File: gatsby-config.js [hunk 5]
```diff
@@ -6,7 +6,7 @@ module.exports = {
 			fullName: `Raghav Sai`,
 		},
 		description: `Raghav Sai's personal site-portfolio-blog.`,
-		siteUrl: `https://www.raghavsai.com/`,
+		siteUrl: `https://raghavsai.com/`,
```
Comment (petgoldfish) [inline gatsby-config.js:9]: Long story, pls no

File: LICENSE [hunk 6]
```diff
@@ -0,0 +1,676 @@
+                    GNU GENERAL PUBLIC LICENSE
```
Comment (petgoldfish) [inline LICENSE:1]: Ummm no pls

**Pull request merged**